src/app/app.module.ts

```diff
@@ -10,8 +10,10 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
 import { AppComponent } from './app.component';
+import { Routing } from './app.router';
```

Comment:
Rename this to RoutingModule to be consistent with Angular's naming conventions.